packages-next/types/src/config/index.ts

```diff
@@ -177,6 +178,13 @@ export type ImagesConfig = {
   };
 };
 
+// config.testing
+
+export type TestingConfig = {
+  isolateRuns?: Boolean;
+  artifactPath?: string;
+};
```

Comment:
What's the reasoning behind having these as part of the keystone config vs arguments to the test stuff?

Reply:
To frame this a bit. I think this is a relatively marginal decision, I could go either way on it. I prefer this as it makes it feel like `testing` is a core part of Keystone, rather than a bolt on (even if that's the case from a technical point of view). This will nudge people to think more about testing when they see it's one of the things which is configurable in their Keystone system (even though almost everyone will be able to use the defaults).